### ISSUE TITLE
[Color, typography and visual elements] Text doesn't meet AA compliance for color contrast. (06.02.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#37072]
#### VFS product name
eGain

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Text doesn't meet AA compliance for color contrast.

### Issue Details
Footer text "Powered by eGain" does not meet color contrast requirements. The "Powered by" text (#b2b2b2) has a contrast ratio of 1.43:1 against the background (#bed8ed). The "eGain" text (#ffffff) has a ratio of 1.47:1 against the background (#bed8ed). In both cases, the contrast ratio should be at least 4.5:1.

### Link, screenshot or steps to recreate
Screenshot: Powered by eGain text with insufficient color contrast.
![powered-by-egain](https://user-images.githubusercontent.com/6844775/154525866-170fde9b-9e9c-4dfa-a7f2-e99fb4bbc030.png)

### VA.gov Experience Standard
[Category Number 06, Issue Number 02](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.4.3_AA
WCAG SC 1.4.6_AAA

---

### Platform Recommendation
Choose higher-contrast color combinations for the footer text, or choose a darker background color for the footer.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.
